src/puya/ir/arc4_router.py

```diff
@@ -60,7 +60,7 @@
 def create_block(
     location: SourceLocation, description: str | None, *stmts: awst_nodes.Statement
```

Comment:
should we propagate the rename of description => comment?

Reply:
In this case, yep.